src/less/chat.less

```diff
@@ -23,6 +24,118 @@
   }
 }
 
+.chat-card-custom {
+  .chat-card-custom-item{
+    img{
+      margin:5px;float:left;
+      border:none;
+      border-radius: 10px;
+      width:30px;
+    }
+
+    p span{
+      display: block;
+      color: black;
+      font-weight: bold;
+    }
+
+    p button {
+      margin: 5px 0px 5px 0px;
+      background-color: indianred;
```

Comment:
Unless there is a stylistic reason for using `indianred` try to use  the predefined `@starfinderRed` value that is defined in `variable.less`.

Reply:
done